docs/concepts/architecture/apiserver-aggregation.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Extending the Kubernetes API with Aggregator
+assignees:
+- chenopis
+---
+
+{% capture overview %}
+
+The *Aggregator* is a system to allow the Kubernetes apiserver to be extended with additional APIs, which are not part of the core Kubernetes APIs. 
```

Comment:
I think the sentence reads a little easier as: "The *Aggregator* is a system that allows the Kubernetes apiserver to be extended with additional APIs, beyond what is offered by the core Kubernetes APIs."?

Let me know if this comment is too fine-grained.